www/site.js

```diff
@@ -0,0 +1,516 @@
+const isOnMobile = window.innerWidth <= 1024;
+
+// The only way we can provide values to wasm_bindgen's generated code is to set globals
+window.js_create_app = js_create_app;
+window.js_run_app = js_run_app;
+window.js_get_result_and_memory = js_get_result_and_memory;
+
+// The only place we use console.error is in wasm_bindgen, where it gets a single string argument.
+console.error = function displayErrorInHistoryPanel(string) {
+  const html = `<div class="panic">${string}</div>`;
+  updateHistoryEntry(repl.inputHistoryIndex, false, html);
+};
+
+import * as roc_repl_wasm from "./repl/roc_repl_wasm.js";
```

Comment:
Do we need the repl js here?

Reply:
Perhaps it's better to wget the main site js with build-dev-local.sh from https://github.com/roc-lang/roc/blob/main/www/public/site.js to prevent them from getting out of sync.

Reply:
Done